src/sentry/static/sentry/app/utils/errorHandler.jsx

```diff
@@ -2,17 +2,32 @@ import React from 'react';
 import RouteError from 'app/views/routeError';
 
 export default function errorHandler(Component) {
-  const originalRender = Component.prototype.render;
```

Comment:
Huh, interesting we were doing this.

Reply:
@EvanPurkhiser it's super old-school React 😄 see: https://github.com/preactjs/preact/issues/531#issuecomment-277789165